### ISSUE TITLE
At official website support.bigbluebutton.org (Close the presentation/enlarge icon)

### ISSUE THREAD

**Issue by konfib** [bigbluebutton/bigbluebutton#16579]
Don't know a better place for report:

**Describe the bug**
At official website (support.bigbluebutton.org)
The "**Close** the presentation/enlarge icon" does not exist anymore
https://support.bigbluebutton.org/hc/en-us/articles/1500005316962-Close-the-presentation-enlarge-the-webcams

**Expected behavior**
To close the presentation, select the "X" (or MINUS)  located in the upper right hand corner of the presentation. 

**Actual behavior**
A three-dot menue, were you cant' close the presentation

**Screenshots**
at the website: https://support.bigbluebutton.org/hc/en-us/articles/1500005316962-Close-the-presentation-enlarge-the-webcams

**BBB version:**
BigBlueButton Server 2.5.11 (3193)

**Comment by antobinary** [comment]
Thanks for reporting this! I have passed the info to the right person to make the necessary tweak!

**Comment by konfib** [comment]
Issue in post 1 still exist.

**Comment by ffdixon** [comment]
Yes, and we'll update the knowledge base with the correct image.
